examples/stress_tests/bevymark_3d.rs

```diff
@@ -0,0 +1,596 @@
+        material_rng: ChaCha8Rng::seed_from_u64(12),
+        velocity_rng: ChaCha8Rng::seed_from_u64(97),
+        transform_rng: ChaCha8Rng::seed_from_u64(26),
+    };
+
+    let font = TextFont {
+        font_size: 40.0,
+        ..Default::default()
+    };
+
+    let camera_distance = (VOLUME_SIZE * 1.3).length();
+    commands.spawn((
+        Camera3d::default(),
+        Transform::from_translation(VOLUME_SIZE * 1.3).looking_at(Vec3::ZERO, Vec3::Y),
+        CameraControl {
+            distance: camera_distance,
+        },
+    ));
+
+    commands.spawn((
+        DirectionalLight {
+            illuminance: 10000.0,
+            shadows_enabled: false,
+            ..default()
+        },
+        Transform::from_xyz(1.0, 2.0, 3.0).looking_at(Vec3::ZERO, Vec3::Y),
+    ));
+
+    commands
+        .spawn((
+            Node {
+                position_type: PositionType::Absolute,
+                padding: UiRect::all(px(5)),
+                ..default()
+            },
+            BackgroundColor(Color::BLACK.with_alpha(0.75)),
+            GlobalZIndex(i32::MAX),
+        ))
+        .with_children(|p| {
+            p.spawn((Text::default(), StatsText)).with_children(|p| {
+                p.spawn((
+                    TextSpan::new("Cube Count: "),
+                    font.clone(),
+                    TextColor(LIME.into()),
+                ));
+                p.spawn((TextSpan::new(""), font.clone(), TextColor(AQUA.into())));
+                p.spawn((
+                    TextSpan::new("\nFPS (raw): "),
+                    font.clone(),
+                    TextColor(LIME.into()),
+                ));
+                p.spawn((TextSpan::new(""), font.clone(), TextColor(AQUA.into())));
+                p.spawn((
+                    TextSpan::new("\nFPS (SMA): "),
+                    font.clone(),
+                    TextColor(LIME.into()),
+                ));
+                p.spawn((TextSpan::new(""), font.clone(), TextColor(AQUA.into())));
+                p.spawn((
+                    TextSpan::new("\nFPS (EMA): "),
+                    font.clone(),
+                    TextColor(LIME.into()),
+                ));
+                p.spawn((TextSpan::new(""), font.clone(), TextColor(AQUA.into())));
+            });
+        });
+
+    let mut scheduled = CubeScheduled {
+        per_wave: args.per_wave,
+        waves: args.waves,
+    };
+
+    if args.benchmark {
+        let counter = counter.into_inner();
+        for wave in (0..scheduled.waves).rev() {
+            spawn_cubes(
+                &mut commands,
+                args,
+                counter,
+                scheduled.per_wave,
+                &mut cube_resources,
+                Some(wave),
+                wave,
+            );
+        }
+        scheduled.waves = 0;
+    }
+    commands.insert_resource(cube_resources);
+    commands.insert_resource(scheduled);
+}
+
+fn camera_control_system(
```

Comment:
We have a built in camera controller now so it should probably be used here.

Reply:
Do we have a built in panorbit? Free cam isn't very useful here, and moving the orbit origin isn't desirable.

Reply:
No built in panorbit yet. We can just cut the camera for now then.